### PR TITLE
[IMP][14.0] barcodes_generator_abstract

### PR DESCRIPTION
**Pull request by AshishHirapara**
updated images path on the configuration readme file (updated to the relative path)

### CODE REVIEW

File: barcodes_generator_abstract/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -4,20 +4,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: /barcodes_generator_abstract/static/description/barcode_rule_tree.png
+.. image:: static/description/barcode_rule_tree.png
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/readme/CONFIGURE.rst]:
```suggestion
.. image:: ../static/description/barcode_rule_tree.png
```

and the rest the same

Comment (AshishHirapara) [reply barcodes_generator_abstract/readme/CONFIGURE.rst]: @pedrobaeza thanks for this suggestion. I have updated the paths. Actually I have used that structure from looking at this https://github.com/OCA/partner-contact/blob/14.0/partner_mobile_duplicate_warn/readme/DESCRIPTION.rst
I will take care next time.

**Review by pedrobaeza — APPROVED** [review]

**Review by francesco-ooops — APPROVED** [review]

**Pull request merged**